### DISCUSSION TITLE
关于 clash skip-auth-prefixes 配置的疑惑

### DISCUSSION THREAD

**Discussion by wanyunr** [MetaCubeX/mihomo#1468] [Q&A]
mihomo 中有一个这样设置
![image](https://github.com/user-attachments/assets/08e8511b-7e72-48b5-9303-3dd8996f8729)
我设置了用户验证，并然后我允许了如下局域网段
```
authentication:
  - "user1:pass1"
skip-auth-prefixes:
  - 127.0.0.0/8
  - ::1/128
  - 192.168.0.0/16
  - 10.0.0.0/8
  - 172.16.0.0/12
  ```
  但是每次打开浏览器都会提示要登录一次
  
![image](https://github.com/user-attachments/assets/de9f237d-410e-489b-bee2-ce3359509687)

  流量应该是从*浏览器* >> 127.0.0.1:7890 >> mihomo，按理说应该是可以跳过验证的，但是无论是在浏览器中使用，还是在Windows中其他sock5中，都会要求验证密码

**Comment by xiandu999-yu** [comment]
插件？

**Reply by wanyunr** [reply]
> 插件？

不是，流量器默认的代理，即使用系统代理

**Reply by xiandu999-yu** [reply]
> > 插件？
> 
> 不是，流量器默认的代理，即使用系统代理

话说你遮端口号干嘛？

**Comment by apoiston** [comment]
这个配置的实现是正常的，可以跳过验证。 我也是这样的网络配置，没有提示要求验证。检查自己的网段、端口配置是否正确。
![image](https://github.com/user-attachments/assets/becc93e2-c50a-47b7-9352-3bf59755df74)
![image](https://github.com/user-attachments/assets/b51a1563-04ed-47e1-bf2c-caa9278665d4)
![image](https://github.com/user-attachments/assets/78f51e7d-a1e5-46e0-90c1-c87b70cb821b)
